src/StripeTests/Services/Terminal/Locations/LocationServiceTest.cs

```diff
@@ -36,10 +36,7 @@ public LocationServiceTest()
                 DisplayName = "my location",
             };
 
-            this.listOptions = new LocationListOptions
-            {
-                OperatorAccount = "acct_123",
-            };
+            this.listOptions = new LocationListOptions();
```

Comment:
ooc why do you prefer the () to the {}? I prefer the latter so that we don't forget to remove the () when we re-add another parameter

Reply:
StyleCop rules force us to have `{}` on different lines so I went for brevity, but I don't care strongly one way or ther other. Updated the PR.